include/ProgramRepresentation/PVAliasSet.h

```diff
@@ -26,6 +26,10 @@ class PVAliasSet {
     // LLVM docs: https://llvm.org/docs/LangRef.html#identifiers
     bool contains(const std::string& cleanedName);
 
+    // returns true iff there is a program var in this set with Value* value.
+    // this overload should only be used if you are looking for value that may
+    // not exist in the branch you're currently analyzing.
```

Comment:
What is the consequence if this overload is used at the wrong time?

Reply:
(same comment applies to the similar text in `ProgramPoint.h`)

Reply:
Well, there would actually be no consequence since it would be as effective as searching by `ProgramVariable`. I wrote "this should only be used ..." at the time to emphasize that this overload was for this one specific bug fix, but I think I will delete that part of the comment. The comment in [`ProgramFunction.h`](https://github.com/Kevin-TD/c-resource-leak-checker/blob/0ab713868045c1d10c2d5a73c01f170d90aadb1e/include/ProgramRepresentation/ProgramFunction.h#L41) for the `getPVASRefFromValue` method explains the situation anyhow